models/public/gmcnn-tf/gmcnn-tf.md

```diff
@@ -0,0 +1,146 @@
+# gmcnn-tf
```

Comment:
I think it would be prudent to call this `gmcnn-places2-tf`, since there are several other GMCNN models in the linked repository.